File: onebot/src/main/java/cn/evole/onebot/sdk/entity/Anonymous.java
package cn.evole.onebot.sdk.entity;

import com.google.gson.annotations.SerializedName;
import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class Anonymous {

    @SerializedName( "id")
    public long id;

    @SerializedName( "name")
    public String name;

    @SerializedName( "flag")
    public String flag;

}

File: onebot/src/main/kotlin/sdk/entity/Anonymous.kt
package cn.evolvefield.onebot.sdk.entity

import com.google.gson.annotations.SerializedName
import lombok.AllArgsConstructor
import lombok.Data

@Data
@AllArgsConstructor
class Anonymous {
    @SerializedName("id")
    var id = 0L
    @SerializedName("name")
    var name: String = ""
    @SerializedName("flag")
    var flag: String = ""
}
